### PR TITLE
Clean: arrow-core

### PR DESCRIPTION
**Pull request by rachelcarmena**
PR on `jo-stacksafe-lazyAp` branch (cc: @1Jajen1 )

### CODE REVIEW

**Comment by rachelcarmena** [comment]
Hi @1Jajen1 !! This PR on your branch solves the problem with the doc generation and validation in #23 :tada:

File: arrow-docs/src/main/kotlin/AQLHelper.kt [hunk 1]
```diff
@@ -20,90 +9,4 @@ typealias BoxOf<A> = arrow.Kind<ForBox, A>
...
-@extension
-interface BoxOrderBy : OrderBy<ForBox> {
-  override fun foldable(): Foldable<ForBox> = Box.foldable()
-}
-
-@extension
-interface BoxUnion : Union<ForBox> {
-  override fun foldable(): Foldable<ForBox> = Box.foldable()
-}
+sealed class Box<out A> : BoxOf<A>
```
Comment (1Jajen1) [inline arrow-docs/src/main/kotlin/AQLHelper.kt]: Is this used somewhere or what is the point keeping this?

File: arrow-docs/src/main/kotlin/ExtensionsHelper.kt [hunk 2]
```diff
@@ -108,47 +107,12 @@ fun String.toKebabCase(): String {
...
-    |${nomnomlMethods()}
-    |${nomnomlHierarchy()}
-  """.trimMargin()
-
-fun TypeClass.nomnomlMethods(): String =
-  "[<typeclass>${kclass.simpleName}|${kclass.declaredFunctions.groupBy { it.name }.toList().joinToString("|") {
-    if (it.second.size > 1) "${it.first}(${it.second.size})"
-    else it.first
-  }}]"
-
```
Comment (1Jajen1) [inline arrow-docs/src/main/kotlin/ExtensionsHelper.kt]: Is this unused or broken? o.O

Comment (rachelcarmena) [reply arrow-docs/src/main/kotlin/ExtensionsHelper.kt]: Unused here

File: arrow-docs/src/main/kotlin/AQLHelper.kt (hunk 1 above)
Comment (rachelcarmena) [reply arrow-docs/src/main/kotlin/AQLHelper.kt]: Used in another point

**Comment by rachelcarmena** [comment]
> Woo a green build ^^ Thanks :)

Thanks to you!! Your PR is super useful to clean Arrow Core :tada:

**Pull request merged**